ozark/pom.xml

```diff
@@ -7,12 +7,53 @@
         <artifactId>parent</artifactId>
         <version>1.0.0-m03-SNAPSHOT</version>
     </parent>
-    <artifactId>ozark</artifactId>
-    <packaging>jar</packaging>
-    <name>Ozark ${project.version} MVC RI</name>
+    <artifactId>javax.mvc</artifactId>
```

Comment:
Hi @lefloh , Im convinced, yes there seems no convention. We can leave the artifact name same, but we can open a discussion on EG list if you want.